### PR TITLE
Modify a compilation problem caused by space in standalone.conf

### PR DESCRIPTION
**Pull request by liangyepianzhou**
Because of the space before and after this, every time I modify standalone.conf and   source  it, it will report an error that the command cannot find on line 593

### CODE REVIEW

File: conf/standalone.conf [hunk 1]
```diff
@@ -590,7 +590,7 @@ managedLedgerDefaultAckQuorum=1
 
 # How frequently to flush the cursor positions that were accumulated due to rate limiting. (seconds).
 # Default is 60 seconds
-managedLedgerCursorPositionFlushSeconds = 60
+managedLedgerCursorPositionFlushSeconds=60
```
Comment (Anonymitaet) [inline conf/standalone.conf:593]: @BewareMyPower do these spaces matter?

Comment (liangyepianzhou) [reply conf/standalone.conf:593]: yes, that's

Comment (liangyepianzhou) [reply conf/standalone.conf:593]: Yes, that's right

Comment (liangyepianzhou) [reply conf/standalone.conf:593]: > @BewareMyPower do these spaces matter?
Yes, that's right

**Comment by codelipenghui** [comment]
@liangyepianzhou Could you please provide more information about the error you have mentioned in the description? The change looks good to me.

**Review by codelipenghui — APPROVED** [review]

**Review by eolivelli — APPROVED** [review]

File: conf/standalone.conf (hunk 1 above)
Comment (BewareMyPower) [reply conf/standalone.conf:593]: I think it doesn't matter, at least on MacOS.

**Review by BewareMyPower — CHANGES_REQUESTED** [review]
For the config format consistency, it's okay.

However, I think your description is ambiguous. Could you provide your detail error logs and your operate system? I tried to modify this config on MacOS like

```properties
managedLedgerCursorPositionFlushSeconds = 120
```

It works well and no error happened. The config was also loaded well (I've added a line to print the config).

Therefore, I think the PR description is not accurate and you need to provide more details.

**Comment by liangyepianzhou** [comment]
Sorry,I haven't  express clearly. This is happened in iTerm with source command.
Such as this:
liangyepianzhou@liangyepianzhoudeMacBook-Pro conf % source standalone.conf
standalone.conf:593: command not found: managedLedgerCursorPositionFlushSeconds

**Comment by BewareMyPower** [comment]
@liangyepianzhou What's the need to run `source conf/standalone.conf`? `conf/standalone.conf` is just the configuration file that is read by pulsar standalone starter. For example,

```bash
bin/pulsar-daemon start standalone
```

It will read `conf/standalone.conf` as the configuration file of pulsar standalone by default.

Or you can run

```bash
PULSAR_STANDALONE_CONF=conf/standalone-2.conf bin/pulsar-daemon start standalone
```

The it will read `conf/standalone-2.conf` as the configuration file of pulsar standalone by default.

`conf/xxx.conf`s are all Java properties files, not a shell file that contains environment variables. Running `source conf/standalone.conf` is meaningless.

In shell script, the variable assignment should not have any space around the `=`:

```bash
VARIABLE=hello # OK
VARIABLE = hello # Error
```

But in Java properties file, spaces around `=` are okay.

**Comment by BewareMyPower** [comment]
Your fix looks like not to be related with your original motivation any more. Would you like to close this PR and open a new PR to describe what's your current fix?

**Pull request closed without merging**